pyspectral/utils.py

```diff
@@ -284,13 +286,13 @@ def download_rsr(dest_dir=None, dry_run=False):
     functions from the internet as tarballs, extracts them, then deletes
     the tarball.
 
-    See :func:`pyspectral.rsr_reader.check_and_download` for a "smart" version
+    See: func: `pyspectral.rsr_reader.check_and_download` for a "smart" version
```

Comment:
Not sure what did this, but this formatting is wrong. `func` is supposed to be surrounded by the colons and right next to the backtick of the function path.

Reply:
What a mess, I think I have reverted back to how it was now. Thanks for spotting this!

Reply:
Yeah, weird. I wouldn't think flake8 would do this and isort shouldn't be touching docstrings.